module/apmlambda/lambda.go

```diff
@@ -92,6 +111,30 @@ func (f *Function) Invoke(req *messages.InvokeRequest, response *messages.Invoke
 	lambdaContext.Request = formatPayload(req.Payload)
 	lambdaContext.Response = ""
 
+	if !ignoreTxnRegistration {
+		defer jsonw.Reset()
+		if err := createPartialTransactionJSON(tx, &jsonw); err != nil {
+			log.Printf("failed to create partial transaction for registration: %v", err)
+		} else {
+			resp, err := http.Post(
+				// TODO: @lahsivjar better way to get base URI
+				"http://localhost:8200/register/transaction",
```

Comment:
Hmm, I'm not sure what the best approach is here. Since it's a special case, I also wouldn't want to complicate the main API.

ELASTIC_APM_SERVER_URLS (multiple URLs) is deprecated, and I think the only time you would set a non-default URL is for testing purposes. So I'd say just use `os.Getenv("ELASTIC_APM_SERVER_URL")` here, and default to `http://localhost:8200`.